_data/meltano/extractors/tap-staffwise/chartica.yml

```diff
@@ -28,6 +28,23 @@ settings:
   label: API Key
   name: api_key
   sensitive: true
+- description: Subdomain you would like to pull data from
+  kind: string
+  label: Subdomain
+  name: subdomain
+- description: Reporting ID of each report
+  kind: password
+  label: Reporting ID
+  name: reporting_id
+  sensitive: true
+- description: The start date for filtering results in the API call
+  kind: string
```

Comment:
I'm curious about the rationale for changing this to `string`.

Reply:
I have checks in the code to see if the values are an empty string and want to avoid any potential errors that might come up if the data type is different.

Reply:
Gotcha